src/server/api/redirect.ts

```diff
@@ -8,7 +8,7 @@ import { UrlCache } from './cache/url'
 import { UrlRepository } from './repositories/url'
 import { DependencyIds } from '../constants'
 import { AnalyticsLogger } from './analytics/analyticsLogger'
-import { CookieReducer } from '../util/transitionPage'
+import { CookieReducer } from '../util/transition-page'
```

Comment:
Can I check why is there a change in naming convention here?

Reply:
this is to maintain consistency with other filenames in our codebase. Check out sequelize-mock.d.ts and box-icon.d.ts